bpy_speckle/operators/commit.py

```diff
@@ -11,19 +11,19 @@
 
 class DeleteCommit(bpy.types.Operator):
     """
-    Deletes the selected commit from the selected stream.
+    Permanently deletes the selected version from the selected model.
     To execute from code, call: `bpy.ops.speckle.delete_commit(are_you_sure=True)`
     """
 
     bl_idname = "speckle.delete_commit"
-    bl_label = "Delete commit"
+    bl_label = "Delete Version"
```

Comment:
I believe we are using lowercase convention except where captialization is needed grammatically